tsup/crypto-shim.js

```
@@ -0,0 +1 @@
+export default window.crypto;
```

ESLint

[error] 'window' is not defined.

(no-undef)